web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -7,8 +7,8 @@ import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/MapControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
-import DestMarker from '@/lib/images/destination-marker.png';
-import BaseMarker from '@/lib/images/base-marker.png';
+import DestMarker from '@/lib/images/destination-marker.txt?raw';
```

Comment:
I had to revert this for now. Unfortunately this breaks in development mode because vite is running on a separate server than the HTML document. I'll make a fix for this in a follow up.